core/store/migrate/migrations/0117_add_log_poller_idx.sql

```diff
@@ -3,5 +3,5 @@ CREATE INDEX logs_idx_block_number ON logs using brin(block_number);
 CREATE INDEX logs_idx_evm_id_event_address_block ON logs using btree (evm_chain_id,event_sig,address,block_number);
 
 -- +goose Down
-DROP INDEX logs_idx_block_number;
-DROP INDEX logs_idx_evm_id_event_address_block;
+DROP INDEX IF EXISTS logs_idx_block_number;
+DROP INDEX IF EXISTS logs_idx_evm_id_event_address_block;
```

Comment:
Is this valid even though we have already released these migrations? (I guess we don't hash them or anything?)

Reply:
yeah all I see from goose is a single integer version corresponding to the migration number